dspace/config/dspace.cfg

```diff
@@ -766,6 +766,22 @@ org.dspace.app.batchitemimport.work.dir = ${dspace.dir}/imports
 # XSL file configured by: crosswalk.dissemination.DataCite.stylesheet file.)
 #identifier.doi.ezid.publisher = a publisher
 
+##### Registry Loader #####
+# Metadata namespaces.
```

Comment:
It would be useful to add a comment that this will import the metadata from ${dspace.dir}/config/registries/ during flyway
I assume this config change should also be documented on the wiki